src/print_table.py

```diff
@@ -0,0 +1,57 @@
+"""This program takes the path to the repo and prints the data table."""
```

Comment:
This file is only used to print out the table of data, correct? I believe that this is beneficial, but does it need to be merged to the `master` branch? I could see the benefit in keeping it, as the CLI team could use it in printing certain information, but we do not want to merge files that are unnecessary (if indeed this is deemed unnecessary by the team).

Reply:
I agree that this file might not be useful for the project as a whole, but it offers an easy way of visualizing and testing the accuracy of the stored data. However, I think that you make a valid point regarding not merging it into `master` branch.